packages/tools/examples/PolySegWasmContourToStackLabelmap/index.ts

```diff
@@ -0,0 +1,302 @@
+
+content.appendChild(viewportGrid);
+
+createInfoSection(content, { ordered: true })
+  .addInstruction('Draw a contour segmentation on the left viewport')
+  .addInstruction(
+    'Click on the button to convert the contour segmentation to a volume labelmap segmentation'
+  );
+
+// ============================= //
+let toolGroup1, toolGroup2;
+let renderingEngine;
+const toolGroupId1 = 'ToolGroup_Contour';
+const toolGroupId2 = 'ToolGroup_Labelmap';
+const viewportId1 = 'CT_SAGITTAL_CONTOUR';
+const viewportId2 = 'CT_SAGITTAL_LABELMAP';
+
+const segmentIndexes = [1, 2, 3, 4, 5];
+
+addButtonToToolbar({
+  title: 'Convert contour segmentation to labelmap segmentation',
+  onClick: async () => {
+    // add the 3d representation to the 3d toolgroup
+    await segmentation.addSegmentationRepresentations(toolGroupId2, [
+      {
+        segmentationId,
+        type: csToolsEnums.SegmentationRepresentations.Labelmap,
+        options: {
+          polySeg: true,
+        },
+      },
+    ]);
+  },
+});
+
+addDropdownToToolbar({
+  labelText: 'Segment Index',
+  options: { values: segmentIndexes, defaultValue: segmentIndexes[0] },
+  onSelectedValueChange: (number) => {
+    segmentation.segmentIndex.setActiveSegmentIndex(
+      segmentationId,
+      Number(number) as number
+    );
+  },
+});
+
+/**
+ * Runs the demo
+ */
+async function run() {
+  // Init Cornerstone and related libraries
+  await initDemo();
+
+  eventTarget.addEventListener(
+    cornerstoneTools.Enums.Events.POLYSEG_CONVERSION_STARTED,
+    (evt) => {
+      const p = document.getElementById('progressDetailP');
+      p.innerHTML = `Conversion started`;
+    }
+  );
+
+  eventTarget.addEventListener(
+    cornerstoneTools.Enums.Events.POLYSEG_CONVERSION_COMPLETED,
+    (evt) => {
+      const p = document.getElementById('progressDetailP');
+      p.innerHTML = `Conversion completed`;
+    }
+  );
+
+  // Add tools to Cornerstone3D
+  cornerstoneTools.addTool(PanTool);
+  cornerstoneTools.addTool(ZoomTool);
+  cornerstoneTools.addTool(StackScrollMouseWheelTool);
+  cornerstoneTools.addTool(SegmentationDisplayTool);
+  cornerstoneTools.addTool(PlanarFreehandContourSegmentationTool);
+  cornerstoneTools.addTool(SegmentSelectTool);
+
+  // Define tool groups to add the segmentation display tool to
+  toolGroup1 = ToolGroupManager.createToolGroup(toolGroupId1);
+  toolGroup2 = ToolGroupManager.createToolGroup(toolGroupId2);
+
+  // Manipulation Tools
+  toolGroup1.addTool(PanTool.toolName);
+  toolGroup1.addTool(ZoomTool.toolName);
+  toolGroup1.addTool(StackScrollMouseWheelTool.toolName);
+  toolGroup1.addTool(PlanarFreehandContourSegmentationTool.toolName);
+  toolGroup1.addTool(SegmentationDisplayTool.toolName);
+  toolGroup1.addTool(SegmentSelectTool.toolName);
+
+  // Segmentation Tools
+  toolGroup2.addTool(PanTool.toolName);
+  toolGroup2.addTool(StackScrollMouseWheelTool.toolName);
+  toolGroup2.addTool(TrackballRotateTool.toolName);
```

Comment:
Not added up above.  Suggest adding trackball to the standard test tools, and removing them from here - should be ok as long as trackball doesn't fail on stack viewports.